src/monster_event.py

```diff
@@ -0,0 +1,47 @@
+"""
+Interfaces for Monster
+"""
+from monster import Monster
+
+from collections import namedtuple
+Equipment = namedtuple('Equipment', ('name', 'strength'))
+Item = namedtuple('Item', ('name', 'heal_value'))
+
+
+class MonsterEvent:
+    """
+    Interfaces class for Monster Event
+    """
+
+    def __init__(self, monster, screen):
+        self.screen = screen
+        self.top = "--- Monster-Event ---"
+        self.header = [
+            "Ein " + monster.name + " " + monster.str + " greift an."
+            ]
+        Monster.fight(monster, player, self);
+
+    def player_win(self, item):
+        self.screen = screen
+        self.top = "--- Monster-Event ---"
+        self.header = [
+            "Du konntest " + monster.name + " besiegen."
+           ]
+        if item != NULL:
```

Comment:
`NULL` gibt es in Python nicht, das ist in Python `None` und sollte mit `is not` verglichen werden statt `!=`